websites/api.py

```diff
@@ -269,6 +269,16 @@ def videos_missing_captions(website: Website) -> List[WebsiteContent]:
     )
 
 
+def draft_content(website: Website) -> List[WebsiteContent]:
+    """Return a list of WebsiteContent objects that are"""
+    if not is_ocw_site(website):
+        return []
```

Comment:
I think you should check with Peter/Ferdi about this. IMO, it makes sense to always issue warnings about drafts.

BTW, re drafts... 
- **in reference to websites**: in the codebase, we refer to "Publish to staging" as publishing the "draft" version of a website
- **in reference to websitecontent:**
     1.  Yaml formatted frontmatter at the start of a markdown file is "metadata" to hugo. Our websitecontent.metadata becomes this yaml-formatted metadata when we publish.
     2. "draft" is a special hugo metadatafield. When you run hugo, it converts all the markdown files to HTML. By default, hugo ignores stuff with `draft: true` in its frontmatter


<img width="792" alt="Screen Shot 2022-08-29 at 11 41 32 AM" src="https://user-images.githubusercontent.com/9010790/187240177-c2dd4859-f2fb-4de4-b0c8-80a7ada6df49.png">

Reply:
> I think you should check with Peter/Ferdi about this. IMO, it makes sense to always issue warnings about drafts.

I agree with Chris.

Reply:
Updated in 6be55ac78b9854334a8c6614ef51965733c893ea.